bot/cogs/trivia.py

```diff
@@ -170,17 +184,55 @@ async def on_message(self, message):
                     embed = discord.Embed(title="Answer")
 
                     if user_answer == correct:
+                        scores[auth_id]+=10
+                        save()
                         values = "You, you got the right answer!"
-                        embed.add_field(name="CORRECT!", value=values, inline=False)
+                        embed.set_footer(text='Your score is: '+str(scores[auth_id]))
                     else:
                         values = "Correct answer was: " + str(correct)
                         embed.add_field(name="INCORRECT!", value=values, inline=False)
                     userAnswers[auth_id] = 0
                     await ctx.send(embed=embed)
+                    await ctx.send('answered?')
 
         except:
             "user not in array"
 
+    #This command prints your scores
+    @commands.command(
+        name="score",
+        brief="returns your score",
+    )
+    async def score(self,ctx):
+      auth_id = str(ctx.message.author.id)
+      if auth_id not in scores:
+        scores[auth_id]=0;
+      embed = discord.Embed(title='Your Score',description=str(scores[auth_id])+' points')
+      await ctx.send(embed=embed)
+
+    @commands.command(
+        name="leaderboard",
+        brief="returns bot trivia leaderboard",
+    )
+    async def leaderboard(self,ctx):
+      leaderboard = {}
+      for f in scores:
+        try:
+          user = await self.bot.fetch_user(int(f))
+          leaderboard[user.display_name] = scores[f]
+        except:
+          "Invalid Author ID"
```

Comment:
this string does not do anything this except can pass, create a message, or raise an exception, but it should not declare a string